client/components/theme-collection/index.tsx

```diff
@@ -144,7 +143,7 @@ export default function ThemeCollection( {
 			<div className="theme-collection__meta">
 				<div className="theme-collection__headings">
 					<h2 className="theme-collection__title">{ title }</h2>
-					<p className="theme-collection__description">{ preventWidows( description ) }</p>
+					<p className="theme-collection__description">{ description }</p>
 				</div>
```

Comment:
Under Appearance > Themes, scroll down to this carousel:

<img width="1168" alt="Screenshot 2025-05-12 at 2 59 03 pm" src="https://github.com/user-attachments/assets/4fc7a47f-2d2a-45d4-8167-40cff009b5ab" />